### ISSUE TITLE
Muting audio/sounds while testing

### ISSUE THREAD

**Issue by Xrayez** [bitwes/Gut#128]
I've come up with this in `gut.gd` to mute all sounds which could be produced while simulating a scene (hearing the same repeating sounds starts to become annoying):
```gdscript
func _init():
	AudioServer.set_bus_volume_db(0, -INF)
```
I wonder whether it's worth to make this as an option.

**Comment by bitwes** [comment]
Maybe.

If you are running from a scene, then it is trivial to add this.  When running from the command line it becomes more difficult.  This makes me think that it might be useful to allow users to hook into the startup of tests in a way that is accessible from both the scene and command line.  I'm not sure what that looks like though.

**Comment by bitwes** [comment]
I've implemented the pre and post hooks in Issue 139 which will allow you to set this at the start of a run.

**Comment by Xrayez** [comment]
Solved by pre-run hook added in 6.8.1, thanks!
```gdscript
extends "res://addons/gut/hook_script.gd"

func run():
    # Mute all sounds during tests
    AudioServer.set_bus_volume_db(0, -INF)
```

.gutconfig.json:
```json
{
    "pre_run_script":"res://tests/pre_run.gd",
}
```
